sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/KeyVaultAccessControlClientImpl.java

```diff
@@ -81,9 +80,7 @@ public RoleAssignmentsImpl getRoleAssignments() {
      */
     public KeyVaultAccessControlClientImpl(String apiVersion) {
         this(
-                new HttpPipelineBuilder()
-                        .policies(new UserAgentPolicy(), new RetryPolicy(), new CookiePolicy())
-                        .build(),
+                new HttpPipelineBuilder().policies(new UserAgentPolicy(), new RetryPolicy()).build(),
```

Comment:
Why is cookie policy removed?

Reply:
I don't know, this change was made by AutoRest. Is this a recent change for our client impls @alzimmermsft, @srnagar?

Reply:
FWIW, I don't know why we'd even want to support cookies for REST API access. The service shouldn't be setting any anyway.

Reply:
Yeah. Removed Cookie after discussed with Srikanta.

I don't know any SDK that really need it. But do let me know if I am wrong.